docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md

```diff
@@ -44,6 +44,27 @@ HRESULT GetColumn (
   
```

Comment:
also if you can:
line 3 - change ms.date to today's date
line 35 - remove two hashes from the parameters heading